dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -87,3 +87,6 @@ lease.check.interval.desc=The interval after which the lease check strategy is i
 
 lease.length=Lease length
 lease.length.desc=The length of time before a recovery log lease expires
+
+lease.renewal.time=Lease Renewal Time
+lease.renewal.time.desc=The percentage of the lease length after which the lease is renewed
```

Comment:
Suggested edit:

`The percentage of the duration of the recovery log lease that passes before the lease is renewed`